app/containers/Root.js

```diff
@@ -208,47 +211,64 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
   }
 }
 
-const Root = ({ history, lnd, onboardingProps, syncingProps }) => {
-  // If we are have not yet onboarded, show the loading/onboarding screen.
-  if (!onboardingProps.onboarding.onboarded) {
-    return (
-      <div>
-        <LoadingBolt
-          theme={onboardingProps.theme}
-          visible={!onboardingProps.onboarding.onboarding}
-        />
-        <Onboarding {...onboardingProps} />
-        <Syncing {...syncingProps} />
-      </div>
-    )
+class Root extends Component {
+  componentWillMount() {
+    const { fetchTicker } = this.props
```

Comment:
don't need to declare a const here if this is the only var we use. can just call `this.props.fetchTicker ()`

Reply:
Our lint rules don't allow for that and not sure that want to change them just for this. Leaving as is for now.